docs/src/Run/README.md

```diff
@@ -1,14 +1,14 @@
-# Running Gateway in Production
+# Running Gateway Services
 
-In this section, you will find instructions to help you get Gateway Services up and running in production.
-Since each of the four Gateway microservices are completely independent of one another, you can begin with whichever service you want or need first.
+In this section, you will find instructions to help you quickly set up Gateway Services. After testing your application with Gateway Services in your local environment, you can proceed to more advanced deployment options.
```

Comment:
I think this introduction is a little more accurate.

Reply:
@mattheworris Perhaps I am missing something, but this feels like how to run the services locally still, not via a production (which I guess is correct per the change in the title).

So that is good, but I think we still need the: I want to run just the account service in [DO/AWS/Rack] via just the Docker image still right?

Reply:
I'd say this looks good, at least as an incremental improvement. If we want more docs on how to spin up in a cloud or other production environment, maybe make another PR?